go-controller/pkg/libovsdb/ops/switch.go

```diff
@@ -328,11 +328,11 @@ func createOrUpdateLogicalSwitchPorts(nbClient libovsdbclient.Client, sw *nbdb.L
 	return err
 }
 
-// CreateOrUpdateLogicalSwitchPortsOnSwitchOps creates or updates the provided
+// CreateOrUpdateLogicalSwitchPortsOnSwitchOpsWithCustomFields creates or updates the provided
 // logical switch ports, adds them to the provided logical switch and returns
 // the corresponding ops
-func CreateOrUpdateLogicalSwitchPortsOnSwitchOps(nbClient libovsdbclient.Client, ops []libovsdb.Operation, sw *nbdb.LogicalSwitch, lsps ...*nbdb.LogicalSwitchPort) ([]libovsdb.Operation, error) {
-	return createOrUpdateLogicalSwitchPortsOps(nbClient, ops, sw, false, lsps...)
+func CreateOrUpdateLogicalSwitchPortsOnSwitchOpsWithCustomFields(nbClient libovsdbclient.Client, ops []libovsdb.Operation, sw *nbdb.LogicalSwitch, customFields []nbdb.LSPCustomField, lsps ...*nbdb.LogicalSwitchPort) ([]libovsdb.Operation, error) {
```

Comment:
Why not just adding a new method?

Reply:
I could keep `CreateOrUpdateLogicalSwitchPortsOnSwitchOps` and add a new `CreateOrUpdateLogicalSwitchPortsOnSwitchOpsWithCustomFields` for our uses but no one will then use the old function, so I decided to replace it entirely.

Would you like me to do so? (= add a new `CreateOrUpdateLogicalSwitchPortsOnSwitchOpsWithCustomFields` and keep the old one `CreateOrUpdateLogicalSwitchPortsOnSwitchOps`?)

Reply:
Ah, ok, so it is only used once. It's an exported method, so its ok to leave it around for others to use in the future or to remove it as you like. But the original method would be the purpose of `getAllUpdatableFields` (o better `getAllNonZeroFields`).

Reply:
OK then I'll remove it.